core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -0,0 +1,315 @@
+package edu.asu.c3simulator.screens;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextField;
+import com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup;
+import com.badlogic.gdx.scenes.scene2d.utils.Align;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.viewport.StretchViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+import edu.asu.c3simulator.simulation.Company;
+import edu.asu.c3simulator.simulation.Employee;
+
+/**
+ * @author Krogstad, Nick
+ * Framework: Moore, Zachary 
+ */
+public class EmploymentScreen implements Screen
+{
+	
+	private class EmployeeListener extends ClickListener
+	{
+		private Employee employee;
+		
+		public EmployeeListener(Employee passedEmployee)
+		{
+			employee = passedEmployee;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			selectedEmployee = this.employee;
+			employeeName.setText("Name: " + employee.getEmployeeName());
+			employeePosition.setText("Position: " + employee.getEmployeePosition());
+			employeePayField.setText("" + employee.getHourlyWage());
+			employeePref.setText("Income Preference: $"
+					+ employee.getPreferredHourlyWage() + " / hr");
+			employeeMorale.setText("Morale: " + employee.getMorale() + " / 10");
```

Comment:
I presume "10" is the maximum moral of an employee.

If so, let's make it a constant in the employee class (public static final int MAXIMUM_MORAL = 10;) and access it here. Also add checks in Employee to ensure the moral never exceeds that number